tools/depends/target/pythonmodule-pil/Makefile

```diff
@@ -1,45 +1,23 @@
 include ../../Makefile.include
-DEPS= ../../Makefile.include Makefile pillow-crosscompile.patch
-
-VERSION.TXT := $(CMAKE_SOURCE_DIR)/version.txt
-APP_NAME=$(shell awk '/APP_NAME/ {print tolower($$2)}' $(VERSION.TXT))
+DEPS= ../../Makefile.include Makefile
 
 # lib name, version
 LIBNAME=Pillow
-VERSION=7.1.2
+VERSION=8.1.0
```

Comment:
is this bump relevant to this PR or should it be pulled out into a separate commit?

Reply:
Not totally, again, long process that included trying to work out existing patch requirements. Always best to get up to date before I go down rabbit holea

Reply:
pulled the bump into its own commit